feature-pack/src/main/resources/modules/system/layers/base/org/apache/httpcomponents/main/module.xml

```diff
@@ -40,5 +38,6 @@
         <module name="org.apache.commons.codec"/>
         <module name="org.apache.commons.logging"/>
         <module name="org.apache.james.mime4j"/>
+        <module name="org.apache.httpcomponents.core"/>
```

Comment:
This change is not equivalent in terms of what the module exposes unless you have exports=true.